src/sps30.cpp

```diff
@@ -543,6 +543,8 @@ uint8_t SPS30::GetValues(struct sps_values *v)
 
     // I2C will only provide valid data bytes depending on I2C buffer
     // if I2C buffer is less than 64 it only providing MASS info (set in constructor)
+
+#if defined INCLUDE_I2C
```

Comment:
This is technically an unrelated change, but `I2C_Max_bytes` is not defined when `INCLUDE_I2C` is not defined.

Reply:
thanks for taking the time and submit. A question/remark: I2C_Max_bytes does NOT  have to be if INCLUDE_I2C is not defined as none of the I2C related code would be included in the driver. But maybe I missed your point, please let me know

Paul

Reply:
Initially I had wired up my sensor incorrectly and toyed a bit with the code to see what was going on. In my efforts I temporarily disabled the `INCLUDE_I2C` flag, and ran into some compilation errors at this piece of code regarding the missing variable. This might however well be because at the time, I was testing on an Arduino Genuino Uno, as I was switching between boards to see what was going on. I see you disable UART for Genuino Uno because it has too little memory to perform anyway, probably the combination of the two flags lead to the compilation error.

I'd happily revert this change as is not mandatory for the SAMD21 modules anyway.

Reply:
Thanks. No change needed. 
If you disable BOTH UART and I2C it will not compile and show an error message
line 46 in SPS30.cpp :
#if !defined INCLUDE_I2C && !defined INCLUDE_UART
#error you must enable either I2C or UART communication
#endif